### ISSUE TITLE
[24H2] When ep_taskbar is active, shortcut global hotkeys do not work

### ISSUE THREAD

**Issue by raymondjpg** [valinet/ExplorerPatcher#3998]
Windows 11 Professional 64 bit 24H2, Explorer Patcher 67.1.

Ctrl_Alt+K works, but Ctrl_Alt+W does not.

Was working with Windows 11 Professional 64 bit 23H2.

Any workaround known?

**Comment by pyrates999** [comment]
did you uninstall EP before upgrading from windows 11 23h2 to windows 11 24h2?

**Comment by raymondjpg** [comment]
> did you uninstall EP before upgrading from windows 11 23h2 to windows 11 24h2?

Yes. The upgrade was offered through Windows Update and stalled at about 25% of installation saying it could not proceed with Explorer Patcher installled. I then uninstalled Explorer Patcher and retried the Windows Update which proceeded to completion. Then installed Explorer Patcher again.

**Comment by raymondjpg** [comment]
> did you uninstall EP before upgrading from windows 11 23h2 to windows 11 24h2?

Yes. The upgrade was offered through Windows Update and stalled at about 25% of installation saying it could not proceed with Explorer Patcher installled. I then uninstalled Explorer Patcher and retried the Windows Update which proceeded to completion. Then installed Explorer Patcher again.

**Comment by pyrates999** [comment]
what taskbar are you using?

**Comment by raymondjpg** [comment]
> what taskbar are you using?

I am using the Windows 10(ExplorerPatcher) Taskbar style.

**Comment by pyrates999** [comment]
and what does Ctrl+Alt+W do when pressed?

**Comment by raymondjpg** [comment]
> and what does Ctrl+Alt+W supposed to do when pressed?

As a desktop shortcut key combination it can be used to fire any shortcut to an executable on the desktop. If you look at the properties of any shortcut on the desktop. the shortcut key combination Ctrl+Alt+W can be set by pressing Ctrl+W together, then OK.

**Comment by pyrates999** [comment]
Works on windows 10 here.

**Comment by pyrates999** [comment]
Open up EP Properties application, and click on restart file explorer in the bottom left corner.

**Comment by raymondjpg** [comment]
Restarting file explorer dies not resolve the issue. Ctrl+Alt+W still does not work with Windows 11 Professional 24H2. It works with native installations of 24H2, also with SrartAllback, just not with Explorer Patcher 67.1.

**Comment by pyrates999** [comment]
Ok, thank you.  Please wait for a developer to respond.

Can you check if any other desktop icon shortcuts that begin with ctrl+alt don't work?  Like ctrl+alt+a, ctrl+alt+b, etc.

**Comment by raymondjpg** [comment]
Ctrl+Alt+A, Ctrl+Alt+B, Ctrl+Alt+X and Ctrl+Alt+Y do not work. I haven't been through the whole alphabet.

(Ctrl+Alt+Z is mapped to another function in the PC, and works with that function.)

**Comment by aufderwiesn** [comment]
I have the same issue.  Just upgraded my OS from Win 11 23H2 to 24H2.  (Uninstalled EP before doing so, then re-installed afterward.)  Using EP version 22621.4317.67.1.  I have several desktop shortcuts to individual files that are located in various folders on my PC.  Each shortcut has a hotkey.  Ctrl-Alt-C works but Ctrl-Alt-P doesn't.  All the hotkeys worked fine when I was still using 23H2.

**Comment by pyrates999** [comment]
Try this.  On the one that doesn't work.  Right click on the shortcut, and select properties, remove the shortcut key and click on the OK button.

Then right click on the shortcut, and select properties, add the shortcut key back and click on the OK button.

**Comment by aufderwiesn** [comment]
I had actually tried that several times yesterday, and it didn't work.  I tried it again just now, with the same result.

But - when I deleted out the hotkey from the Properties tab - and then did a re-start on the PC - and THEN added the hotkey back, it worked.  Funny, I thought I tried this yesterday but maybe not.  In any case, thank you for leading me to the solution.  Hopefully it will stick.

It also seems like re-starts are taking much longer now than when I was still using 23H2.  Don't know if this is a Windows issue or an EP issue.  But I can live with it.

**Comment by aufderwiesn** [comment]
Spoke too soon.  I just logged off, logged on again, and the hotkey is again not working.

**Comment by pyrates999** [comment]
Ok, please wait for a developer to respond.

**Comment by Amrsatrio** [comment]
Confirmed 26100.2314 ARM64. Fix will come this month.

**Comment by Amrsatrio** [comment]
Identified the issue in ep_taskbar codebase.

**Comment by Amrsatrio** [comment]
Fixed in 68.2.

**Comment by raymondjpg** [comment]
Thanks for the update.
